.ocp-indent

```diff
@@ -1,5 +1,4 @@
 normal
-max_indent = 2
-strict_with = auto
 align_ops = false
 align_params = never
```

Comment:
These are unecessary because if they are ever useful (eg. breaking the line inside an expression), OCamlformat will reformat the whole thing differently anyway.